src/cloud/project/routes.md

```diff
@@ -87,7 +83,36 @@ https://refactorcss-mswy7hzcuhcjw.us.magentosite.cloud/blog
   {:.no-copy}
 
 {:.bs-callout-info}
-{{site.data.var.ece}} also supports [multiple applications]({{ site.baseurl }}/cloud/project/project-multi-sites.html) per project. Each project has a single `routes.yaml` file that defines which request is routed to which application.
+If your Cloud project supports multiple stores, follow the route configuration instructions for [multiple websites or stores]({{ site.baseurl }}/cloud/project/project-multi-sites.html).
+
+## Route protocols
+
+All environments support both HTTP and HTTPS automatically.
+
+- If the configuration specifies only the HTTP route, HTTPS routes are created automatically, allowing the site to be served from both HTTP and HTTPS without requiring redirects. For example, if you have project with the default domain `example.com`, the record `http://{default}/` resolves to the following URLs:
+
+   ```text
+   http://example.com/
+
+   https://example.com/
+   ```
+   {:.no-copy}
+
+- If the configuration specifies only the HTTPS route, then all HTTP requests redirect to HTTPS. For example, for the default domain `example.com`, the route `https://{default}/` resolves to URL `https://example.com/` and redirects `http://example.com/` to `https://example.com/`.
+
+What is similar to
```

Comment:
As discussed, the phrase "What is similar to" and example can be deleted as the preceding bullet point explains the concept.

Reply:
done